### PR TITLE
Add config to support disabling admin pages by default.

### PR DESCRIPTION
**Pull request by onurhunce-planetly**
https://github.com/django-waffle/django-waffle/issues/443

### CODE REVIEW

File: docs/starting/configuring.rst [hunk 1]
```diff
@@ -101,3 +101,6 @@ behavior.
     e.g. ``logging.WARNING``.
     Defaults to ``None``.
     
+
+``WAFFLE_ADMIN_PAGES_ARE_ENABLED``
```
Comment (clintonb) [inline docs/starting/configuring.rst]: `WAFFLE_ENABLE_ADMIN_PAGES`

File: docs/starting/configuring.rst [hunk 2]
```diff
@@ -101,3 +101,6 @@ behavior.
...
+    Admin pages can be disabled via this configuration. It is True by default.
```
Comment (clintonb) [inline docs/starting/configuring.rst]: Enables the default admin pages for Waffle models. This is `True` by default, but can be disabled to override or customize the pages.

**Review by clintonb — CHANGES_REQUESTED** [review]
I have suggestions for improving the naming and description. Otherwise, this is great!

File: docs/starting/configuring.rst (hunk 2 above)
Comment (onurhunce-planetly) [reply docs/starting/configuring.rst]: Done ✅

File: docs/starting/configuring.rst (hunk 1 above)
Comment (onurhunce-planetly) [reply docs/starting/configuring.rst]: Done ✅

**Review by clintonb — APPROVED** [review]

**Pull request merged**